gmso/core/angle.py

```diff
@@ -73,6 +87,24 @@ def restraint(self):
         """Return the restraint of this angle."""
         return self.__dict__.get("restraint_")
 
+    @property
+    def bonds(self):
+        """Return the bond_order symbol of this bond."""
+        return self.__dict__.get("bonds_")
+
+    @bonds.setter
+    def bonds(self, bonds):
+        """Return the bonds that makeup this Improper.
```

Comment:
I think these doc strings aren't correct for this setter method?

Reply:
remove setter.